package.json

```diff
@@ -42,9 +52,16 @@
     "build-min": "cross-env NODE_ENV=production webpack",
     "build-watch": "webpack --watch",
     "test": "karma start --single-run",
-    "prepublish": "npm run build"
+    "prepublish": "npm run build",
+    "lint": "eslint --ext .js src/"
   },
   "lint-staged": {
-    "*.{js,css,md}": "prettier --write"
+    "*.{js,css,md}": [
+      "prettier --write",
+      "eslint --fix"
+    ],
+    "package.json": [
+      "sort-package-json"
```

Comment:
another small question is, why we added the "sort-package-json" here, can you share what is the intention with this addition here =)? we want to have as few dependencies as possible

Reply:
It's to organize package.json.
[sort-package-json](https://www.npmjs.com/package/sort-package-json)

---
Example:
```
$ cd my-project
$ cat package.json
{
  "dependencies": {
    "sort-package-json": "1.0.0",
    "sort-object-keys": "1.0.0"
  },
  "version": "1.0.0",
  "name": "my-awesome-project"
}

$ npx sort-package-json
package.json is sorted!

$ cat package.json
{
  "name": "my-awesome-project",
  "version": "1.0.0",
  "dependencies": {
    "sort-object-keys": "1.0.0",
    "sort-package-json": "1.0.0"
  }
}
```

Reply:
ahh ok I see, but still, I guess is not part of the lint proposal, and I do think that we should avoid unnecessary deps into the project guess we should remove it before merging

Reply:
sure. i already remove this dependence.